config/tags.php

```diff
@@ -117,11 +117,8 @@
 					return $img;
 				}
 
-				if ($link = $tag->file($tag->link)) {
-					$link = $link->url();
-				} else {
-					$link = $tag->link === 'self' ? $tag->src : $tag->link;
-				}
+				$link   = $tag->file($tag->link)?->url();
+				$link ??= $tag->link === 'self' ? $tag->src : $tag->link;
```

Comment:
Same here.

Reply:
@bastianallgeier @texnixe @lukasbestle While I can see in the first case that a one-liner makes sense, here that would get really hard to read. I really think the `??=` syntax makes a lot of sense here, maybe just something for you to get used to as `??=` is rather new to see for us?

Reply:
> I really think the `??=` syntax makes a lot of sense here, maybe just something for you to get used to as `??=` is rather new to see for us?

I agree. 👍